specification/network/resource-manager/Microsoft.Network/stable/2023-05-01/firewallPolicy.json

```diff
@@ -838,7 +838,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/firewallPolicies/{firewallPolicyName}/listIdpsFilterOptions": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/firewallPolicies/{firewallPolicyName}/intrusionDetectionSignaturesOverrides": {
```

Comment:
Does this match the controller? This also seems like a permissions change

Reply:
yes, the old value was wrong